NCU-Project/Users/UsersViewModel.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  UsersViewModel.swift
+//  NCU-Project
+//
+//  Created by Simon Rowlands on 19/10/2018.
+//  Copyright © 2018 simonrowlands. All rights reserved.
+//
+
+import Foundation
+
+import RxSwift
+import RxCocoa
+
+final class UsersViewModel: ViewModelType {
+    
+    struct Input {
+        let postsButtonTap: ControlEvent<Void>
```

Comment:
instead of `ControlEvent`, pass `Observable` and in view controller do 
`let input = UsersViewModel.Input(postsButtonTap: getPostsButton.rx.tap.asObservable(), userButtonTap: getUserButton.rx.tap.asObservable())`